### PR TITLE
feature - get team

### PR DESCRIPTION
**Pull request by ghost**

### CODE REVIEW

File: Demo/Model/API/API.Team.swift [hunk 1]
```diff
@@ -0,0 +1,30 @@
...
+//  Copyright © 2017 Asian Tech Co., Ltd. All rights reserved.
+//
+
+import Foundation
+import Alamofire
+import ObjectMapper
+
+extension Api.Team {
+    struct QueryParams {
+        let teamId: Int
```
Comment (at-daonguyen) [inline Demo/Model/API/API.Team.swift]: Please rename to `id`.

File: Demo/Model/Schema/Team.swift [hunk 2]
```diff
@@ -0,0 +1,36 @@
...
+import RealmS
+
+final class Team: Object, StaticMappable {
+    dynamic var id = 0
+    dynamic var name = ""
+    dynamic var slug = ""
+    dynamic var desc: String?
+    dynamic var memberCount = 0
+    dynamic var repoCount = 0
+    override static func primaryKey() -> String? {
```
Comment (at-daonguyen) [inline Demo/Model/Schema/Team.swift]: Please break newline.

File: Demo/View/GitTeamViewController.swift [hunk 3]
```diff
@@ -0,0 +1,51 @@
...
+//  Demo
+//
+//  Created by Huynh Quang Tien on 5/26/17.
+//  Copyright © 2017 Asian Tech Co., Ltd. All rights reserved.
+//
+
+import UIKit
+import MVVM
+
+class GitTeamViewController: UIViewController, MVVM.View {
```
Comment (at-daonguyen) [inline Demo/View/GitTeamViewController.swift]: This is `final` class, right? What does `Git` mean?

File: Demo/ViewModel/Team/TeamDetailViewModel.swift [hunk 4]
```diff
@@ -0,0 +1,27 @@
...
+    var desc: String?
+    var memberCount = ""
+    var repoCount = ""
+
+    init(team: Team?) {
+        guard let team = team else { return }
+        name = team.name
+        slug = team.slug
+        desc = team.desc
+        memberCount = "\(team.memberCount)会員"
```
Comment (at-daonguyen) [inline Demo/ViewModel/Team/TeamDetailViewModel.swift]: What is `会員`?

File: Demo/ViewModel/Team/TeamViewModel.swift [hunk 5]
```diff
@@ -0,0 +1,44 @@
...
+//
+
+import Foundation
+import RealmSwift
+import RealmS
+import MVVM
+
+final class TeamViewModel: MVVM.ViewModel {
+    let teamId: Int
+    var teamDetailViewModel: TeamDetailViewModel? {
```
Comment (at-daonguyen) [inline Demo/ViewModel/Team/TeamViewModel.swift]: Please break newline.

**Review by at-daonguyen — CHANGES_REQUESTED** [review]
@at-tienhuynh Please check my comments, thanks you!

File: Demo/ViewModel/Team/TeamDetailViewModel.swift (hunk 4 above)
Comment (ghost) [reply Demo/ViewModel/Team/TeamDetailViewModel.swift]: This point is example for format and present member in UILabel, `会員` is mean `member` in Japan.

File: Demo/Model/API/API.Team.swift (hunk 1 above)
Comment (ghost) [reply Demo/Model/API/API.Team.swift]: Fixed

File: Demo/Model/Schema/Team.swift (hunk 2 above)
Comment (ghost) [reply Demo/Model/Schema/Team.swift]: Fixed!

File: Demo/View/GitTeamViewController.swift (hunk 3 above)
Comment (ghost) [reply Demo/View/GitTeamViewController.swift]: Fixed!

File: Demo/ViewModel/Team/TeamViewModel.swift (hunk 5 above)
Comment (ghost) [reply Demo/ViewModel/Team/TeamViewModel.swift]: Fixed!

File: Demo/ViewModel/Team/TeamDetailViewModel.swift (hunk 4 above)
Comment (at-daonguyen) [reply Demo/ViewModel/Team/TeamDetailViewModel.swift]: Why do you use Japanese here?

**Review by at-daonguyen — CHANGES_REQUESTED** [review]
@at-tienhuynh Please check my comments.

**Review by at-daonguyen — APPROVED** [review]
LGTM!

**Pull request merged**